<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgAPUChannels</class>
 <widget class="QWidget" name="wdgAPUChannels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>206</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>APU Channels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_channels">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>active</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBox_Triangle">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSlider" name="horizontalSlider_Noise">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_Square1">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="Line" name="line_Triangle">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_value_slider_Triangle">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>volume</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_value_slider_Noise">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBox_DMC">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="Line" name="line_Square2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSlider" name="horizontalSlider_DMC">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSlider" name="horizontalSlider_Square2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_value_slider_Square1">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_Master">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_Triangle">
       <property name="text">
        <string>Triangle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="Line" name="line_Square1">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="Line" name="line_Master">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_value_slider_Extra">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="horizontalSlider_Master">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="Line" name="line_Extra">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="Line" name="line_Noise">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_value_slider_DMC">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_Noise">
       <property name="text">
        <string>Noise</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_value_slider_Square2">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkBox_Noise">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_Square2">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_DMC">
       <property name="text">
        <string>DMC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_Extra">
       <property name="text">
        <string>Extra</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Square1">
       <property name="text">
        <string>Square 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_value_slider_Master">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSlider" name="horizontalSlider_Triangle">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSlider" name="horizontalSlider_Extra">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkBox_Extra">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSlider" name="horizontalSlider_Square1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="Line" name="line_DMC">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Square2">
       <property name="text">
        <string>Square 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Master">
       <property name="text">
        <string>Master</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_APU_Channels">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="pushButton_APU_Channels_active_all">
       <property name="text">
        <string>Active all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_APU_Channels_disable_all">
       <property name="text">
        <string>Disable all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_APU_Channels_reset">
     <property name="text">
      <string>Reset to defaults</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icon/icons/overscan_default.svg</normaloff>:/icon/icons/overscan_default.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_Master</tabstop>
  <tabstop>horizontalSlider_Master</tabstop>
  <tabstop>checkBox_Square1</tabstop>
  <tabstop>horizontalSlider_Square1</tabstop>
  <tabstop>checkBox_Square2</tabstop>
  <tabstop>horizontalSlider_Square2</tabstop>
  <tabstop>checkBox_Triangle</tabstop>
  <tabstop>horizontalSlider_Triangle</tabstop>
  <tabstop>checkBox_Noise</tabstop>
  <tabstop>horizontalSlider_Noise</tabstop>
  <tabstop>checkBox_DMC</tabstop>
  <tabstop>horizontalSlider_DMC</tabstop>
  <tabstop>checkBox_Extra</tabstop>
  <tabstop>horizontalSlider_Extra</tabstop>
  <tabstop>pushButton_APU_Channels_active_all</tabstop>
  <tabstop>pushButton_APU_Channels_disable_all</tabstop>
  <tabstop>pushButton_APU_Channels_reset</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
